app/templates/macros/nav_macros.html

```diff
@@ -19,6 +19,9 @@
 
 {% macro account_items(current_user) %}
     {% if current_user.is_authenticated() %}
+        <div class="ui small inverted transparent icon input">
+            <input type="text" placeholder="Search members...">
```

Comment:
What are we planning on searching for here (members, resources)? I know it's just a placeholder, but I want to make sure the purpose is clear. Also, we should probably give it an id. Perhaps `id="search-nav"`.